### DISCUSSION TITLE
CARDDAV error ModSecurity

### DISCUSSION THREAD

**Discussion by testmodem** [YetiForceCompany/YetiForceCRM#14346] [General]
Dear all I am trying to configure Yetiforce to sync contacts from my mobile to Yetiforce-contacts. 
I get error on my mobile davx5 app attached below.
[davx5-error.docx](https://github.com/YetiForceCompany/YetiForceCRM/files/6098402/davx5-error.docx)

and some error from error.log = 
[apache2-error.log.docx](https://github.com/YetiForceCompany/YetiForceCRM/files/6098403/apache2-error.log.docx)

Thanks

**Comment by mariuszkrzaczkowski** [comment]
ModSecurity is blocking communication
"Request content type is not allowed by policy"

**Reply by testmodem** [reply]
Thanks for your reply.
Can you please guide me toward any documentation / Howto to enable the required configuration.

Your help will be much appreciated

**Reply by mariuszkrzaczkowski** [reply]
Try to textually disable mod_security  
https://www.google.com/search?q=mod_security+disable

**Reply by testmodem** [reply]
Thanks .. Ill work on it for some hours.  lets hope i get it resolved.

dell@ubuntu:/etc/modsecurity$ apache2 -v
Server version: Apache/2.4.46 (Ubuntu)
Server built:   2020-08-25T12:13:38

**Reply by testmodem** [reply]
> Try to textually disable mod_security
> https://www.google.com/search?q=mod_security+disable

Thanks got it working . Edited apache2.conf and  disabled SecRuleEngine and SecRequestBodyAccess Off.

But this time I am getting error due to Bday field in vcard while syncing . how can i change its format to DD-MM-YYYY ?


FN:Test1
--
TEL;CELL:00441111111111
TEL;CELL:00441111111111
EMAIL;X-INTERNET:abc@abc.com
BDAY:1983-02-22
END:VCARD
BEGIN:VCARD
VERSION:2.1


Thanks

**Reply by mariuszkrzaczkowski** [reply]
create a new issue
where do you generate the ics file?

**Reply by testmodem** [reply]
Ics file ? sorry i am really a starterm trying to get it working through following different tutorials.

https://github.com/YetiForceCompany/YetiForceCRM/discussions/14364

New issue created (:
